test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -1310,6 +1363,64 @@ def check_if_sz_vav(model, zone)
     return zone_system_check
   end
 
+  # Check if preheat coil control for system 5 through 8 are implemented
+  #
+  # @param baseline_base [Hash] Baseline
+  def check_preheat_coil_ctrl(baseline_base)
+    baseline_base.each do |baseline, model_baseline|
+      building_type, template, climate_zone, mod = baseline
+
+      # Concatenate modifier functions and arguments
+      mod_str = mod.flatten.join('_') unless mod.empty?
+
+      htg_coil_node_list = []
+      model_baseline.getAirLoopHVACs.each do |airloop|
+        # Baseline system type identified based on airloop HVAC name
+        if airloop.name.to_s.include?('Sys5') ||
+            airloop.name.to_s.include?('Sys6') ||
+            airloop.name.to_s.include?('Sys7') ||
+            airloop.name.to_s.include?('Sys8')
+
+          # Get all Heating Coil in the airloop.
+          heating_coil_outlet_node = nil
+          airloop.supplyComponents.each do |equip|
+            if equip.to_CoilHeatingWater.is_initialized
+              htg_coil = equip.to_CoilHeatingWater.get
+              heating_coil_outlet_node = htg_coil.airOutletModelObject.get.to_Node.get
+            elsif equip.to_CoilHeatingElectric.is_initialized
+              htg_coil = equip.to_CoilHeatingElectric.get
+              heating_coil_outlet_node = htg_coil.outletModelObject.get.to_Node.get
+            elsif equip.to_CoilHeatingGas.is_initialized
+              htg_coil = equip.to_CoilHeatingGas.get
+              heating_coil_outlet_node = htg_coil.airOutletModelObject.get.to_Node.get
```

Comment:
Similar comment as above, I think that if the system type is 5 or 7 we should fail the test as it should be hydronic.

Reply:
Add this as a failure case - for system 5-8